libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -825,6 +919,23 @@ void AP_Baro::update(void)
             }
             float altitude = sensors[i].altitude;
             float corrected_pressure = sensors[i].pressure + sensors[i].p_correction;
+            // correct for ststic pressure position errors
+            Vector3f airspeed_vec_bf;
```

Comment:
we need
#if !defined(HAL_BUILD_AP_PERIPH) around this.
I also think we should make this a new function, maybe wind_compensation() ?